source/exercises/detrend.md

```diff
@@ -1,3 +1,72 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.0
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# 去线性趋势
+
+- 本节贡献者: {{何星辰}}、{{田冬冬}}、{{姚家园}}
+- 最近更新日期: 2025-10-21
+- 预计花费时间: 10 分钟
+
+---
+
+去线性趋势是指计算出一条最能拟合整个波形数据的直线（即趋势线），并从每个数据点中减去该直线对应的值，
+使波形围绕稳定的零基线振动。此操作用于去除由温度变化或地壳形变等因素造成的缓慢漂移与基线偏移。
+
+我们以前一节使用的 2022 年 9 月 22 日墨西哥 Mw 6.8 地震在 ANMO 台站的波形为例。
+
+```{code-cell} ipython3
+from obspy import UTCDateTime
+from obspy.clients.fdsn import Client
+import matplotlib.pyplot as plt
+from numpy.polynomial import polynomial
```

Comment:
Following the example in https://numpy.org/doc/stable/reference/generated/numpy.polynomial.polynomial.polyfit.html#numpy.polynomial.polynomial.polyfit.
```suggestion
from numpy.polynomial import polynomial as P
```